docs/credentials.adoc

```diff
@@ -1,16 +1,130 @@
-= cerdentials.zip file format
+= credentials.zip file format
 
-To authenticate on the ostree server, aktualizr uses cerdentials.zip archive.
+The credentials.zip file provides the link between a user's on-line account on ATS Garage or OTA+ deployment and developer's machine. It is used for:
 
-The minimal requirement for this archive is including `treehub.json` file with
-the following content:
+* Uploading the built OSTree image to treehub with garage-push
+* Offline-signing Uptane images targets metadata with garage-sign
+* Provisioning the device with Aktualizr or aktualizr_cert_provider
+* Providing the URL to the update server to Aktualizr (this is templated into sota.toml at build time, to allow software updates to modify that URL)
 
+The following files are present in credentials.zip:
+
+[options="header"]
+|======================
+| Filename in zip | Purpose | Used by
+| treehub.json | Location and authentications for treehub and Uptane repo | garage-sign, garage-push
+| client.crt | Certificate for TLS client authentication | Aktualizr 
+| client.key | Private key for TLS client authentication | Aktualizr
+| root.crt | Root CA for TLS client authentication | Aktualizr
+| repo.url
```

Comment:
This is what I see:
```
tufrepo.url
autoprov.url
autoprov_credentials.p12
root.json
targets.pub
targets.sec
treehub.json
```
`client.crt`, `client.key`, and `root.crt` are all part of `autoprov_credentials.p12`.